packages/nc-gui-v2/composables/useViewData.ts

```diff
@@ -114,14 +116,13 @@ export function useViewData(
 
   const loadData = async (params: Parameters<Api<any>['dbViewRow']['list']>[4] = {}) => {
     if ((!project?.value?.id || !meta?.value?.id || !viewMeta?.value?.id) && !isPublic.value) return
-
     const response = !isPublic.value
-      ? await $api.dbViewRow.list('noco', project.value.id!, meta.value.id!, viewMeta!.value.id, {
-          ...params,
-          ...(isUIAllowed('sortSync') ? {} : { sortArrJson: JSON.stringify(sorts.value) }),
-          ...(isUIAllowed('filterSync') ? {} : { filterArrJson: JSON.stringify(nestedFilters.value) }),
-          where: where?.value,
-        })
+      ? await api.dbViewRow.list('noco', project.value.id!, meta.value.id!, viewMeta!.value.id, {
```

Comment:
if we are using `api` from `useApi`, I believe all `$api` from `useNuxtApp` should also be replaced as well?

Reply:
If the api call is supposed to trigger `isLoading`, then yea, `$api` should be replaced. :)

Reply:
Only for the list API we need the spinner